src/CattyTests/Scene/SceneTests.swift

```diff
@@ -58,123 +58,123 @@ final class SceneTests: XCTestCase {
     }
 
     func testTouchConversionCenter() {
-        let scaledScene = SceneBuilder(project: ProjectMock(width: self.screenSize.width * 2, andHeight: self.screenSize.height * 2)).build()
+        let scaledStage = StageBuilder(project: ProjectMock(width: self.screenSize.width * 2, andHeight: self.screenSize.height * 2)).build()
         let scaledSceneCenter = CGPoint(x: self.screenSize.width / 2, y: self.screenSize.height / 2)
```

Comment:
please rename this variable & following scaledScene.... 
and there are some variables (Line 37-41) in this file also named with scene
and please also rename this file and the parent folder